hack/deps.sh

```diff
@@ -24,15 +29,15 @@ function dispatch() {
 
   shift 2
   local specified_targets="$*"
-  if [[ -n ${specified_targets} ]] && [[ ! ${specified_targets} =~ ${target} ]]; then
+  if [[ -n ${specified_targets} ]] && [[ ${specified_targets} != "update" ]] && [[ ! ${specified_targets} =~ ${target} ]]; then
```

Comment:
How do we use the new update target?

Reply:
`make deps update` -- upgrade the go modules